src/bindings/bnd_surface.cpp

```diff
@@ -58,6 +77,8 @@ void initSurfaceBindings(void*)
     .function("isSingular", &BND_Surface::IsSingular)
     .function("isAtSingularity", &BND_Surface::IsAtSingularity)
     .function("isAtSeam", &BND_Surface::IsAtSeam)
+    .function("toNurbsSurface", &BND_Surface::ToNurbsSurfaceDefault)
```

Comment:
@pilcru I think you just need to add an `, allow_raw_pointers()` parameter here to get the WASM CI build to work